ui/tests/acceptance/stepDefinitions/accountsContext.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert')
```

Comment:
Codacy found an issue: ['assert' is assigned a value but never used.](https://app.codacy.com/gh/owncloud/ocis-accounts/pullRequest?prid=5799018)